src/sentry/db/models/base.py

```diff
@@ -74,7 +76,10 @@ def _update_tracked_data(self):
             data = {}
             for f in self._meta.fields:
                 try:
-                    data[f.column] = self.__get_field_value(f)
+                    if isinstance(self.__get_field_value(f), BitHandler):
```

Comment:
Last nit: We don't need to call this `__get_field_value` more than once.

So just maybe something like:

```python
try:
    v = self.__get_field_value(f)
except AttributeError as e:
    logging.exception(six.text_type(e))
else:
   if isinstance(v, BitHandler):
       v = copy(v)
   data[f.column] = v
```

Reply:
👍 that works, updated